### PR TITLE
jaccard similarity implemented

### PR DESCRIPTION
**Pull request by nithinkrishna**

### CODE REVIEW

**Comment by harsham05** [comment]
@nithinkrishna I see that your PR computes **pairwise** Jaccard Distance. Absolute Jaccard Distance/Golden Feature Set was already implemented & is used as a baseline

**Comment by nithinkrishna** [comment]
@harsham05 Ah. Got it. Some comments on top of similarity.py would've helped :)

File: jaccard_similarity.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        union = len(f1MetaData.keys()) + len(f2MetaData.keys()) - intersection
+        jaccard = float(intersection) / union
+
+        a.writerow([file1, file2, jaccard])
+
+
+if __name__ == "__main__":
+
+    argParser = argparse.ArgumentParser('Cosine similarity based on Metadata values')
```
Comment (chrismattmann) [inline jaccard_similarity.py]: shouldn't this read Jaccard similarity?

**Comment by chrismattmann** [comment]
happy to merge this as another option. see my comments. Also make sure this doesn't stomp out the other jaccard absolute distance.

**Comment by nithinkrishna** [comment]
@chrismattmann I've made the fix :)

**Comment by chrismattmann** [comment]
thank you @nithinkrishna !

**Pull request merged**